src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -439,7 +439,7 @@ const ContextMenuActions: ContextMenuAction[] = [
                     setClipboardMessage(CONST.ACTIONABLE_TRACK_EXPENSE_WHISPER_MESSAGE);
                 } else if (ReportActionsUtils.isRenamedAction(reportAction)) {
                     setClipboardMessage(ReportActionsUtils.getRenamedAction(reportAction));
-                } else if (reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.SUBMITTED) {
+                } else if (ReportActionsUtils.isActionOfType(reportAction, CONST.REPORT.ACTIONS.TYPE.SUBMITTED)) {
```

Comment:
Hmm I might need to update this to copy the "automatically..." text, not just "submitted..." 🙃

Reply:
fixing in https://github.com/Expensify/App/pull/49909